Module4.3_Search_and_Sorting_Algorithms/activities/Leet Code Activity/1.md

```diff
@@ -0,0 +1,30 @@
+### Sorting a Linked List 
+
+The goal for this activity is to sort an unsorted linked list by using the **Insertion Sort** algorithm. 
+
+![Insertion Sort](https://i.ytimg.com/vi/OGzPmgsI-pQ/maxresdefault.jpg)
+
+As a reminder, insertion sort divides a list into two parts: the sorted part and the unsorted part. The algorithm starts off with the first two elements in the list, and sorts them. This will be the sorted part of the list while the rest is the unsorted part. Then as it traverses the list, it will add another element to the sorted part of the list and resort it. This will continue until the whole list is sorted. 
+
+Here are some examples of what our solution will do:
+
+**Example 1:**
+
+```
+Input: 4->2->1->3
```

Comment:
formatting

Reply:
kind of confused, what's wrong with the formatting?

Reply:
the spatial difference after the colons for the input and the output